GOVERNANCE.md

```diff
@@ -1,114 +1,295 @@
+* Selecting the Code of Conduct Committee and ratifying CoC judgements
+* Removing projects which have become inactive
+* Acting on escalated security or code quality issues
+* Resolving issues that individual projects are unable to resolve
+* Administering project infrastructure, intellectual property, and resources
+* Determining overall direction for advocacy and marketing
+* Issuing statements on behalf of Operator Framework and its subprojects
+* Reviewing and approving Contributor Ladder advancement for participants who
+  work on the overall umbrella project
+* Facilitate outreach and project participation in CNCF events
+
+In performance of these duties, the Steering Committee will hold a
+monthly meeting that is open to all contributors. The Committee may hold
+additional, closed meetings in order to discuss non-public issues such
+as security exploits, CoC enforcement, and legal questions.
+
+Steering Committee members are expected to advocate for all of Operator
+Framework, not just certain projects or corporate sponsors, comply with
+and support the Code of Conduct, and be professional and compassionate
+in all of their dealings with project participants.
+
+### Steering Committee Elections
+
+Once per year, the Steering Committee will select three Election
+Officers to run the annual election and set the dates for the election.
+Election Officers should be project Members in good standing, not
+running for election themselves, and represent more than one project
+employer. These officers will update the list of eligible Members according to
+project records, send out announcements, and conduct the election.
+
+The election itself will last for at least one week, and is conducted as
+a preference election online, using the Condorcet method. The Election
+Officers will announce the selected candidates at the next regular
+community meeting.
+
+#### Steering Committee Election Procedure
+
+1. Nominations for Operator Framework Representatives (Subproject Rep
+   and Community Rep) will be collected for at least 14 days. Interested
+   individuals should self-nominate, and they may be considered for
+   Subproject Representative and Community Representative.
+2. For each Subproject, one Subproject Representative is chosen by the
+   Subproject's Approvers. Each Subproject vote will be conducted by the
+   OF Election Officials and calculated by the Condorcet Method.
+3. Following the announcement of the Subproject results, the
+   Community Representatives are chosen by all Members of the Operator
+   Framework. The vote will be conducted by the OF Election Officials
+   and calculated by the Condorcet Method.
+
+If individuals are nominated to be Subproject Representative, the Subproject elections will be calculated in a random order. No individual can
+hold more than one seat on the Steering Committee.
+
+### Steering Committee Bootstrapping
+
+Upon ratification of this document, the 7 elected Chairs of the Operator
```

Comment:
This is not really clear to me, so the current 7 chairs will be steering committee for the next year?

Reply:
yes, thats correct. FWIW, if we adopt a new Subproject, they will also get a rep, so we would have 8 or more